### PR TITLE
How to create, rename, and delete project

### PR DESCRIPTION
**Pull request by dewan-ahmed**
# What changed, and why it matters

Migration article for https://help.aiven.io/en/articles/5039826-how-to-create-new-project and https://help.aiven.io/en/articles/4758669-renaming-a-project. This PR also adds the information on deleting a project.

### CODE REVIEW

**Comment by dewan-ahmed** [comment]
Note to reviewer: please add the relevant entries to the migration sheet or let me know and I can do it. Thanks.

File: docs/platform/howto/manage-project.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+Renaming a project is possible **only** when all the services in the project are powered-off. To rename a project in the `Aiven web console <https://console.aiven.io/>`_:
+
+1. Make sure the respective project is selected from the **Project** drop down.
+2. Click on **Settings**. 
+3. Type in the new project name in the *Project Name* field.
+4. Click on **Save changes**. 
+
+.. note:: 
+   
+   - Except for Aiven for Apache Kafka®, all other service types have backups which are restored once you power them back on.
```
Comment (staceysalamon-aiven) [inline docs/platform/howto/manage-project.rst:22]:
```suggestion
   - Except for Aiven for Apache Kafka®, all service types have backups which are restored once you power them back on.
```

File: docs/platform/howto/manage-project.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+1. Click on **Project** drop down and click on **Create new project**.
+2. Give the new project a name.
+3. When you create a new project, you will need to enable billing for creating new services.
+
+Rename a project
+================
+
+Renaming a project is possible **only** when all the services in the project are powered-off. To rename a project in the `Aiven web console <https://console.aiven.io/>`_:
+
+1. Make sure the respective project is selected from the **Project** drop down.
```
Comment (staceysalamon-aiven) [inline docs/platform/howto/manage-project.rst:15]: Just for clarity and readability, I would reword this to: Select the project from the **Project** drop down.

But up to you!

File: docs/platform/howto/manage-project.rst [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+   - Except for Aiven for Apache Kafka®, all other service types have backups which are restored once you power them back on.
+   - Renaming the project will invalidate all of the project's pending invitations.
+   - The project name in your existing DNS records will not be updated.
+
+Delete a project
+================
+
+In order to delete a project, you need to removes the services in it first. Once all the services are removed:
+
+1. Make sure the respective project is selected from the **Project** drop down.
```
Comment (staceysalamon-aiven) [inline docs/platform/howto/manage-project.rst:31]: Same suggestion as above. :)

File: docs/platform/howto/manage-project.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+2. Click on **Settings**.
+3. Click on **Remove project**. 
+
+.. note::
+    You can :ref:`create, rename and delete a project using the Aiven CLI <avn-manage-project>` as well.
```
Comment (staceysalamon-aiven) [inline docs/platform/howto/manage-project.rst:36]: Since this relates to all operations (create, rename and delete), I think we should move it to the beginning of the article.

You could also add in the introduction for this article that the instructions on this page are all for the Aiven Console. This would then also mean that you could remove the "in the Aiven web console" part in the Create and Rename sections.

**Review by staceysalamon-aiven — CHANGES_REQUESTED** [review]
Made a few suggestions and added some thoughts in the comments.

**Comment by staceysalamon-aiven** [comment]
> Note to reviewer: please add the relevant entries to the migration sheet or let me know and I can do it. Thanks.

Sorry, do I just need to update the Article Migrated column in the spreadsheet?

**Comment by dewan-ahmed** [comment]
> Sorry, do I just need to update the Article Migrated column in the spreadsheet?

Hi, info provided in DM :)

**Review by staceysalamon-aiven — APPROVED** [review]
Looks good!
